dist/components/tabs/readme.md

```diff
@@ -0,0 +1,53 @@
+# Tabs Component
+
+There are two parts with tabs component: tabs controls and tabs body.
+Tabs controls list all tab buttons, placed in one line. Tabs body is tabs
+content in tabs body.
+
+
+## Installation
+
+Refer to the main [installation instructions for Stencil](https://github.com/mobify/stencil#installation).
+
+Use the following to import the Tabs Component into your project with Sass:
+
+```
+    @import '../bower_components/stencil/dist/components/tabs';
```

Comment:
Looks like this was based off the old version of the readme template which has since been updated. The import path should be `@import '../bower_components/mobify-stencil/dist/components/tabs/tabs';`